jetstream/fakespot-in-suggest.toml

```diff
@@ -0,0 +1,203 @@
+[metrics]
+weekly = ["admarketplace_sponsored_impressions",
+          "fakespot_amazon_impressions",
+          "fakespot_bestbuy_impressions",
+          "fakespot_walmart_impressions",
+          "fakespot_amazon_impression_rate",
+          "fakespot_bestbuy_impression_rate",
+          "fakespot_walmart_impression_rate",
+          "fakespot_amazon_clicks",
+          "fakespot_bestbuy_clicks",
+          "fakespot_walmart_clicks",
+          "fakespot_amazon_ctr",
+          "fakespot_bestbuy_ctr",
+          "fakespot_walmart_ctr",
+          "fakespot_amazon_not_relevant_dismissals",
+          "fakespot_bestbuy_not_relevant_dismissals",
+          "fakespot_walmart_not_relevant_dismissals",
+          "fx_urlbar_quick_suggest_remote_settings_latency_ms"]
+
+overall = ["admarketplace_sponsored_impressions",
+           "fakespot_amazon_impressions",
+           "fakespot_bestbuy_impressions",
+           "fakespot_walmart_impressions",
+           "fakespot_amazon_impression_rate",
+           "fakespot_bestbuy_impression_rate",
+           "fakespot_walmart_impression_rate",
+           "fakespot_amazon_clicks",
+           "fakespot_bestbuy_clicks",
+           "fakespot_walmart_clicks",
+           "fakespot_amazon_ctr",
+           "fakespot_bestbuy_ctr",
+           "fakespot_walmart_ctr",
+           "fakespot_amazon_not_relevant_dismissals",
+           "fakespot_bestbuy_not_relevant_dismissals",
+           "fakespot_walmart_not_relevant_dismissals",
+           "fx_urlbar_quick_suggest_remote_settings_latency_ms"]
+
+## Impressions
+[metrics.admarketplace_sponsored_impressions]
+select_expression = "COUNT(DISTINCT IF(is_terminal AND result.result_type like '%adm_sponsored', event_id, NULL))"
```

Comment:
Is there a way to re-use this UDF? https://github.com/mozilla/bigquery-etl/blob/main/sql/mozfun/norm/result_type_to_product_name/udf.sql

Reply:
for the impressions, we don't currently map the product name to the results array, only to the selected result. this poses a problem when you want to calculate the impression since these have the original name and not the remapped values. 
I am not aware of a way to use the udf here to keep the business logic in one place and keep this DRY